File: pinyougou-manager-web/src/main/webapp/js/service/contentCategoryService.js
//服务层
app.service('contentCategoryService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../contentCategory/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../contentCategory/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../contentCategory/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../contentCategory/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../contentCategory/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../contentCategory/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../contentCategory/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: pinyougou-content-interface/src/main/java/com/pinyougou/content/service/ContentCategoryService.java
package com.pinyougou.content.service;
import java.util.List;
import com.pinyougou.pojo.TbContentCategory;

import entity.PageResult;
/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface ContentCategoryService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbContentCategory> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageResult findPage(int pageNum,int pageSize);
	
	
	/**
	 * 增加
	*/
	public void add(TbContentCategory contentCategory);
	
	
	/**
	 * 修改
	 */
	public void update(TbContentCategory contentCategory);
	

	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbContentCategory findOne(Long id);
	
	
	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long [] ids);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbContentCategory contentCategory, int pageNum,int pageSize);
	
}
